Refactor(eos_cli_config_gen, eos_designs)!: Make the type key as required for aaa_accounting (#5881)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

      ipv4_acl: IPV4
```

### Required keys for eos_designs data models

> The following data model keys have been required from `eos_designs` in AVD 6.0.0.

Reviewer: ```suggestion
Starting AVD 6.0.0, the following data model keys have been made required in `eos_designs`.
```

> | Required key |
> | ----------- |

Reviewer: ```suggestion
| Required keys |
| ------------- |
```

+     - name: tapagg
```

### Required keys for eos_cli_config_gen data models

> The following data model keys have been required from `eos_cli_config_gen` in AVD 6.0.0.
> 
> | Required key |
> | ----------- |

Reviewer: ```suggestion
Starting AVD 6.0.0, the following data model keys have been made required in `eos_cli_config_gen`.

| Required keys |
| ------------- |
```
